src/test/common.ts

```diff
@@ -139,10 +139,11 @@ async function setGlobalPathToInterpreter(pythonPath?: string): Promise<void> {
     await pythonConfig.update('defaultInterpreterPath', pythonPath, true);
     await disposePythonSettings();
 }
-export async function disableExperimentsInPythonExtension(): Promise<void> {
+export async function adjustSettingsInPythonExtension(): Promise<void> {
     const vscode = require('vscode') as typeof import('vscode');
     const pythonConfig = vscode.workspace.getConfiguration('python', (null as any) as Uri);
-    await pythonConfig.update('experiments.enabled', false, vscode.ConfigurationTarget.Global).then(noop, noop);
+    await pythonConfig.update('logging.level', 'debug', vscode.ConfigurationTarget.Global);
```

Comment:
This shoudln't be necessary, we've set an environment variable at the top of the YAML file.

Reply:
This is forcing it for python, not jupyter.

Reply:
Wait, I added the env variable in our YAML, all you need is `VSC_PYTHON_FORCE_LOGGING` and that enabled debugigng logging, simpler than updating this setting.

Reply:
Hmm, can't find that, i'm certain I added that, anyways

Reply:
I'd suggest adding that along with out setting `VSC_JUPYTER_FORCE_LOGGING: 'true'`